src/Sylius/Behat/Context/Ui/Admin/NotificationContext.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Ui\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\NotificationType;
+use Sylius\Behat\Service\NotificationCheckerInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+final class NotificationContext implements Context
+{
+    /**
+     * @var NotificationCheckerInterface
+     */
+    private $notificationChecker;
+
+    /**
+     * @param NotificationCheckerInterface $notificationChecker
+     */
+    public function __construct(NotificationCheckerInterface $notificationChecker)
+    {
+        $this->notificationChecker = $notificationChecker;
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully created
+     */
+    public function iShouldBeNotifiedItHasBeenSuccessfullyCreated()
+    {
+        $this->notificationChecker->checkNotification('has been successfully created.', NotificationType::success());
```

Comment:
I would leave it as it is now. There is one edge case when we could benefit from that, there could be a notification `has not been successfully created.` 💥  :trollface: